programs/wen_new_standard/src/instructions/mint/freeze.rs

```diff
@@ -0,0 +1,83 @@
+use anchor_lang::prelude::*;
+
+use anchor_spl::token_interface::{
+    Mint, Token2022, TokenAccount,
+    FreezeAccount, freeze_account,
+};
+use spl_pod::solana_program::program_option::COption;
+
+use crate::{Manager, MANAGER_SEED, MintErrors};
+
+#[derive(Accounts)]
+pub struct FreezeDelegatedAccount<'info> {
+    #[account(mut)]
+    pub payer: Signer<'info>,
+    #[account()]
+    /// CHECK: can be any account
+    pub user: UncheckedAccount<'info>,
+    #[account(mut)]
+    pub delegate_authority: Signer<'info>,
+    #[account()]
```

Comment:
remove

Reply:
Sorry @balmy-gazebo what should i remove? User?